### ISSUE TITLE
operator-no-newline-before and operator-no-unspaced fail with CSS Custom Properties

### ISSUE THREAD

**Issue by kaelig** [kristerkari/stylelint-scss#252]
According to my tests, these two rules:

```
"scss/operator-no-newline-before": true,
"scss/operator-no-unspaced": true
```

Fail to lint:

```
:root {
  --foo: '{{}}';
}
```

Trace:

```
TypeError: Cannot read property 'isCalculationEnabled' of undefined
    at findOperators (/Users/kaelig/src/github.com/Shopify/polaris-react/node_modules/stylelint-scss/dist/utils/sassValueParser/index.js:77:37)
    at /Users/kaelig/src/github.com/Shopify/polaris-react/node_modules/stylelint-scss/dist/rules/operator-no-unspaced/index.js:150:45
    at /Users/kaelig/src/github.com/Shopify/polaris-react/node_modules/postcss-scss/node_modules/postcss/lib/container.js:144:26
    at Rule.each (/Users/kaelig/src/github.com/Shopify/polaris-react/node_modules/postcss-scss/node_modules/postcss/lib/container.js:110:22)
    at Rule.walk (/Users/kaelig/src/github.com/Shopify/polaris-react/node_modules/postcss-scss/node_modules/postcss/lib/container.js:143:21)
    at /Users/kaelig/src/github.com/Shopify/polaris-react/node_modules/postcss-scss/node_modules/postcss/lib/container.js:146:32
    at Root.each (/Users/kaelig/src/github.com/Shopify/polaris-react/node_modules/postcss-scss/node_modules/postcss/lib/container.js:110:22)
    at Root.walk (/Users/kaelig/src/github.com/Shopify/polaris-react/node_modules/postcss-scss/node_modules/postcss/lib/container.js:143:21)
    at calculationOperatorSpaceChecker (/Users/kaelig/src/github.com/Shopify/polaris-react/node_modules/stylelint-scss/dist/rules/operator-no-unspaced/index.js:143:8)
    at /Users/kaelig/src/github.com/Shopify/polaris-react/node_modules/stylelint-scss/dist/rules/operator-no-newline-before/index.js:17:61
```

**Comment by kristerkari** [comment]
Thanks! I need verify, but looks like a bug :)

**Comment by YozhikM** [comment]
I confirm the error is caused with this CSS.
```bash
at findOperators (src/utils/sassValueParser/index.js:154:207)
      at src/rules/operator-no-unspaced/index.js:139:45
      at node_modules/stylelint/node_modules/postcss-scss/node_modules/postcss/lib/container.es6:109:26
      at Rule.each (node_modules/stylelint/node_modules/postcss-scss/node_modules/postcss/lib/container.es6:77:22)
      at Rule.walk (node_modules/stylelint/node_modules/postcss-scss/node_modules/postcss/lib/container.es6:108:21)
      at node_modules/stylelint/node_modules/postcss-scss/node_modules/postcss/lib/container.es6:111:32
      at Root.each (node_modules/stylelint/node_modules/postcss-scss/node_modules/postcss/lib/container.es6:77:22)
      at Root.walk (node_modules/stylelint/node_modules/postcss-scss/node_modules/postcss/lib/container.es6:108:21)
      at calculationOperatorSpaceChecker (src/rules/operator-no-unspaced/index.js:132:8)
      at src/rules/operator-no-newline-before/index.js:55:61
```

**Comment by kristerkari** [comment]
@YozhikM That would be great, thanks for the help!

**Comment by kristerkari** [comment]
Fix available in v3.1.1. Thanks for helping!

**Comment by kaelig** [comment]
Thank you!
